Yoast/wordpress-seo
# Optimize recalculation

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* All post types should be calculated in one progressbar like process.

## Test instructions

This PR can be tested by following these steps:

* Do a `grunt build:js`
* Have some post that needs to be index.
* See everything still working.

⚠️  Make sure to have #7319 merged first. ⚠️
